plugins/gatsby-remark-dvc-linker/commandLinker.js

```diff
@@ -5,31 +5,29 @@ const { getItemByPath } = require('../../src/utils/shared/sidebar')
 
 const DVC_REGEXP = /dvc\s+[a-z][a-z-.]*/
 const COMMAND_REGEXP = /^[a-z][a-z-]*$/
+const ARGS_REGEXP = new RegExp(/\-{1,2}[a-zA-Z-]*/, 'ig')
 const COMMAND_ROOT = '/doc/command-reference/'
 
 module.exports = astNode => {
   const node = astNode[0]
   const parent = astNode[2]
-
   if (parent.type !== 'link' && DVC_REGEXP.test(node.value)) {
     const parts = node.value.split(/\s+/)
+    const baseUrl = `${COMMAND_ROOT}${parts[1]}`
```

Comment:
Not 100% what changed in this plugin to cause this but this phrase in [`/config`](https://dvc-org-interactive-syn-s47nhu.herokuapp.com/doc/command-reference/config#--system) is now a broken link ([in dvc.org](https://dvc.org/doc/command-reference/config#command-options-flags), this phrase isn't a link at all):

![image](https://user-images.githubusercontent.com/43496356/148588938-147510e6-f022-4641-b745-9986d09271a6.png)

Reply:
We have updated the command linker as per the requirement. I have now fixed the logic and the above command should be linkable as `/config#--system`.